testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/management/cli/DeployAllDomainTestCase.java

```diff
@@ -60,42 +63,27 @@
     protected static String sgOne;
     protected static String sgTwo;
 
-    protected CommandContext ctx;
+    protected static CommandContext ctx;
     protected static DomainTestSupport testSupport;
+    protected static DeploymentInfoUtils infoUtils;
 
     @BeforeClass
     public static void before() throws Exception {
-
         testSupport = CLITestSuite.createSupport(UndeployWildcardDomainTestCase.class.getSimpleName());
-
-        String tempDir = System.getProperty("java.io.tmpdir");
+        infoUtils = new DeploymentInfoUtils(DomainTestSupport.masterAddress);
```

Comment:
As I commented in DeploymentInfoUtils class, I think that we shouldn't have DeploymentInfoUtils class to deal with connection to CLI.